Formula/x/xml2rfc.rb

```diff
@@ -0,0 +1,151 @@
+class Xml2rfc < Formula
+  include Language::Python::Virtualenv
+
+  desc "Tool to convert XML RFC7749 to the original ASCII or the new HTML look-and-feel"
+  homepage "https://xml2rfc.tools.ietf.org/"
+  url "https://files.pythonhosted.org/packages/19/5a/4e4cbd0d669ff6bd368ba185bfbad3fb91ac4d500374398d2e245ddc5394/xml2rfc-3.28.1.tar.gz"
+  sha256 "76201b28f5b5d4c466dad74d972b3260013db0f462c76ebc633ea1c7fcd87194"
+  license "BSD-3-Clause"
+  head "https://github.com/ietf-tools/xml2rfc.git", branch: "main"
+
+  depends_on "python@3.12"
+  uses_from_macos "libxml2", since: :ventura
+  uses_from_macos "libxslt"
+
+  on_linux do
+    depends_on "libxslt"
+  end
+  resource "certifi" do
+    url "https://files.pythonhosted.org/packages/e8/9e/c05b3920a3b7d20d3d3310465f50348e5b3694f4f88c6daf736eef3024c4/certifi-2025.4.26.tar.gz"
+    sha256 "0a816057ea3cdefcef70270d2c515e4506bbc954f417fa5ade2021213bb8f0c6"
+  end
+
+  resource "charset-normalizer" do
+    url "https://files.pythonhosted.org/packages/e4/33/89c2ced2b67d1c2a61c19c6751aa8902d46ce3dacb23600a283619f5a12d/charset_normalizer-3.4.2.tar.gz"
+    sha256 "5baececa9ecba31eff645232d59845c07aa030f0c81ee70184a90d35099a0e63"
+  end
+
+  resource "ConfigArgParse" do
```

Comment:
there are resource case issues, which I think you can resolve by running `brew update-python-resources xml2rfc`

Reply:
Sounds good, 👍